client/src/client-only-routes/show-certification.tsx

```diff
@@ -389,6 +389,14 @@ const ShowCertification = (props: ShowCertificationProps): JSX.Element => {
 
   const isMicrosoftCert = certSlug === Certification.FoundationalCSharp;
 
+  const isLanguageCert = [
```

Comment:
We can use this constant instead:

https://github.com/freeCodeCamp/freeCodeCamp/blob/c147f726f53cfa0e78ff3d7a199df5108b83c367/shared/config/curriculum.ts#L47-L54